ci-operator/config/openshift-service-mesh/istio/openshift-service-mesh-istio-master.yaml

```diff
@@ -153,21 +153,22 @@ tests:
           export DOCKER_INSECURE_REGISTRIES="default-route-openshift-image-registry.$(oc get routes -A -o jsonpath='{.items[0].spec.host}' | awk -F. '{print substr($0, index($0,$2))}')" && \
           export BUILD_WITH_CONTAINER="0" && \
           export CONTROL_PLANE_SOURCE=sail &&\
+          export INSTALL_SAIL_OPERATOR=true &&\
           cd /work && \
           entrypoint \
           prow/integ-suite-ocp.sh \
-          telemetry"
+          telemetry 'TestStatsGatewayServerTCPFilter|TestDashboard/pilot-dashboard.json|TestStatsGatewayServerTCPFilter|TestStatsTCPFilter/additional-labels|TestStatsFilter/additional-labels|TestStatsFilter/mockprom-to-metrics|TestServerTracing/primary-0|TestClientTracing/primary-0|TestBadWasmRemoteLoad|TestStatsGatewayServerTCPFilter'"
```

Comment:
I prefer to not skip working tests, but instead get into fixing the converted failures.